client/src/components/Header/Header.test.js

```diff
@@ -249,7 +249,7 @@ const hasRadioNavItem = component => {
 };
 
 const hasSignOutNavItem = component => {
-  const { children } = navigationLinks(component, 10);
+  const { children } = navigationLinks(component, 12);
```

Comment:
Is it possible to avoid magic numbers in code for readability? What does 12 represent?

Reply:
The 12th child of the NavLink component tree.

Yes I think the tests could be potentially modified to look for the child by text content instead.